python/ccxt/base/exchange.py

```diff
@@ -2769,7 +2769,7 @@ def handle_market_type_and_params(self, method_name, market=None, params={}):
             if isinstance(method_options, str):
                 method_type = method_options
             else:
-                method_type = self.safe_string_2(method_options, 'defaultType', 'type')
+                method_type = self.safe_string_2(method_options, 'defaultType', 'type', default_type)
```

Comment:
can we make the python the same as the other langs

Reply:
ouch, mechanical mistake of mine.